sdk/highlight-grafana-datasource/src/README.md

```diff
@@ -0,0 +1,49 @@
+<!-- This README file is going to be the one displayed on the Grafana.com website for your plugin. Uncomment and replace the content here before publishing.
```

Comment:
could we have a description of highlight / the plugin for publishing in the future?

Reply:
👍 will get a PR out tomorrow with this + grafana datasource tests